src/walletgen/Walletgen.js

```diff
@@ -19,8 +19,8 @@ import Done from './views/Done';
 
 import './style/index.css';
 
-const checkNetwork = () =>
-  navigator.onLine ? NETWORK_STATES.ONLINE : NETWORK_STATES.OFFLINE;
+// TODO: actually merge these two projects into one
```

Comment:
Not entirely sure what's meant by this.

Reply:
it's referencing a file from bridge directly, which breaks the "these two projects are totally separate" assumption, but that will change once wallet generation is an actual feature of bridge